src/sql/src/session/vars.rs

```diff
@@ -1714,6 +1714,11 @@ feature_flags!(
         "the COMMENT ON feature for objects",
         false, // default false
         false  // internal false
+    ),
+    (
+        enable_unified_optimizer_api,
+        "use the new unified optimzier API in bootstrap() and coordinator methods",
+        true
```

Comment:
This is going to default to on? Fine with that just making sure.

Reply:
> This is going to default to on?

Yes, the idea is to roll this out and just have the switch as an option to disable so we can revert the changes without cutting a new release if we see something wrong.

Reply:
As discussed in the team sync, in its current form this will be on unconditionally during bootstrapping.